### PR TITLE
Relnotes 0.17.10

### PR DESCRIPTION
**Pull request by mackrorysd**
## Description

This is mostly the original release notes, plus a few capitalization fixes, etc. The DB SSL support in Helm charts got modified just to refer people to the values.yaml file for more technical details.

### CODE REVIEW

**Comment by mackrorysd** [comment]
@dzhu I don't remember this including non-release-note diffs last time. Have I missed a step, I wonder?

**Comment by gh-determined-ai** [comment]
See release notes comments for PR#3690.

File: docs/release-notes.txt [hunk 1]
```diff
@@ -8,6 +8,34 @@
...
+**Release Date:** March 03, 2022
+
+**Breaking Change**
+
+-  API: Pytorch Lightning has been updated from 1.3.5 to 1.5.9 to address a security vulnerability.
+   Experiments using Pytorch Lightning Adapter with v1.3.5 are no longer supported.
+
+**New Features**
+
+-  Added self-supervised image classification example in PyTorch using Bootstrap Your Own Latent
```
Comment (vishnu2kmohan) [inline docs/release-notes.txt]:
```suggestion
-  Added a Bootstrap Your Own Latent (BYOL) self-supervised image classification PyTorch example.
```

File: docs/release-notes.txt [hunk 2]
```diff
@@ -8,6 +8,34 @@
...
+   (BYOL).
+
+-  PyTorchTrial and TFKerasTrial now automatically log the number of batches and number of records
+   in every training and validation workload, as well as the duration of the workload and the
+   calculated batches per second and records per second.
+
+-  All logs produced by tasks on the cluster are persisted and accessible through the new ``det task
```
Comment (vishnu2kmohan) [inline docs/release-notes.txt]:
```suggestion
-  All (experiment & non-experiment) task logs are persisted & accessible through the new ``det task
```

File: docs/release-notes.txt [hunk 3]
```diff
@@ -8,6 +8,34 @@
...
+   logs`` CLI command or the usual commands to retrieve their logs, even after completion.
+
+-  Now supports setting root certificates for the DB via the Determined Helm chart. This allows
```
Comment (vishnu2kmohan) [inline docs/release-notes.txt]:
```suggestion
-  Support specifying root certificates for the DB via the Determined Helm chart. This allows
```

**Review by vishnu2kmohan — APPROVED** [review]
Small suggestions, but LGTM. Thanks @mackrorysd (and @gh-determined-ai)!

**Comment by mackrorysd** [comment]
I'm confused by the existence of both https://github.com/determined-ai/determined/pull/3690 and https://github.com/determined-ai/determined/pull/3690 and this at the same time...

**Pull request closed without merging**